client/views/room/MessageList/lib/isMessageSequential.ts

```diff
@@ -1,15 +1,14 @@
-import { differenceInMinutes } from 'date-fns';
+import { differenceInSeconds } from 'date-fns';
 
+import { MessageTypes } from '../../../../../app/ui-utils/client';
 import { IMessage } from '../../../../../definition/IMessage';
 
-// TODO: use settings config to define time range instead of hardcoded value
-
-export const isMessageSequential = (current: IMessage, previous: IMessage | undefined): boolean => {
+export const isMessageSequential = (current: IMessage, previous: IMessage | undefined, groupingRange: number): boolean => {
```

Comment:
maybe a default value to groupingRage?

Reply:
Hmmm, I think we don't intend this function to be called without this groupingRange, specially because it depends on a setting. Since there's no real use for it outside this scope, I think it's better to narrow it's use as much as possible.